cmd/tether/ops_linux.go

```diff
@@ -146,22 +147,12 @@ func (t *operations) SetupFirewall(config *tether.ExecutorConfig) error {
 	// calls wait and attempts to collect its child, but the reaper will
 	// have raptured the pid before that.  So, best effort, just keep going.
 	_ = netfilter.Flush(context.Background(), "")
-
-	// default rule set
-	established := &netfilter.Rule{
-		Chain:  netfilter.Input,
-		States: []netfilter.State{netfilter.Established},
-		Target: netfilter.Accept,
-	}
-
-	reject := &netfilter.Rule{
-		Chain:  netfilter.Input,
-		Target: netfilter.Reject,
-	}
+	generalPolicy(netfilter.Drop)
+	configuredExternal := false
 
 	for _, endpoint := range config.Networks {
-		if endpoint.Network.Type == constants.ExternalScopeType {
-
+		if endpoint.Network.Type == constants.ExternalScopeType && !configuredExternal {
```

Comment:
Previously, there was a "break" statement at the end of any external scope type. I was just replicating that behavior, but guaranteeing that all of the bridges are set up as well. I will remove the "configuredExternal" flag.